LoRaEngine/LoraKeysManagerFacade/EUIValidator.cs

```diff
@@ -4,40 +4,23 @@
 namespace LoraKeysManagerFacade
 {
     using System;
+    using LoRaWan;
 
-    internal static class EUIValidator
+    internal static class EuiValidator
     {
-        private const string InvalidZero = "0000000000000000";
-
         /// <summary>
         /// DevEUI are required to be IEEE EUI-64.
         /// </summary>
-        /// <param name="devEUI">devEUI to validate.</param>
+        /// <param name="devEui">devEUI to validate.</param>
         /// <exception cref="ArgumentException">If an invalid devEUI string was passed.</exception>
-        internal static void ValidateDevEUI(string devEUI)
+        internal static bool TryParseAndValidate(string devEui, out DevEui result)
```

Comment:
Isn't this doing the same as `DevEui.TryParse` and therefore can be entirely replace with it?

Reply:
Not entirely. It also validates that the result is not equal to the `default`, for which we had a test case. Instead of removing the test case and using `TryParse` directly I decided to keep this method. We will fail faster if we assign a `default` `DevEui` somewhere on the Network Server side this way.

Reply:
I see. Perhaps we can then turn this into a one-liner and use `IsValid` from `DevEui` (which also treats `FFFF:FFFF:FFFF:FFFF` as invalid per spec)?

```c#
internal static bool TryParseAndValidate(string devEui, out DevEui result) =>
    DevEui.TryParse(devEui, out result) && result.IsValid;
```

nit: Eventually I think this could be an option on `TryParse` of `DevEui`. The goal here would be to avoid individual floating methods and have related functionality living closer.

Reply:
Addressed this with 7e8d334d92f7136d27b191026084332193fc5632.